JobConfig/cosmic/prolog.fcl

```diff
@@ -103,15 +109,19 @@ Cosmic: {
    }
    dsResample: {
```

Comment:
Why does a cosmic job have a module named 'dsResample'?  Was this just a copy from Andrei's example?  I think a name like cosmicResampler would be more intuitive.